docs/resources/rule_set_response_header.md

```diff
@@ -3,19 +3,19 @@ layout: "wallarm"
 page_title: "Wallarm: wallarm_rule_set_response_header"
 subcategory: "Rule"
 description: |-
-  Provides the "Specify a response header" rule resource.
+  Provides the "Change server response headers" rule resource.
 ---
 
 # wallarm_rule_set_response_header
 
 !> The resource requires additional permissions. Ask the support team to obtain them.
 
-Provides the resource to manage rules with the "Specify a response header" action type. This rule type is used to configure supplementary headers appended or replaced by a Wallarm WAF node.
+Provides the resource to manage rules with the "[Change server response headers][1]" action type. This rule type is used to configure supplementary headers appended or replaced by a Wallarm node.
```

Comment:
it also allows to delete headers

Reply:
@aoleynikov-wallarm, can this resource delete server response headers? And if so, then how (can you give an example)?

Reply:
Well, according to [the docs](https://docs.wallarm.com/user-guides/rules/add-replace-response-header/#creating-and-applying-the-rule):

> To delete an existing response header, please specify a space as the value of this header on the Replace tab.

You can do the same thing using the provider.

```hcl
resource "wallarm_rule_set_response_header" "delete_header" {
  mode = "replace"

  headers = {
    X-FOOBAR = " "
  }
}
```